apps/marketplace/pages/DownloadReports/DownloadReports.js

```diff
@@ -0,0 +1,213 @@
+          '': {
+            startDateIsValid,
+            endDateIsValid,
+            startAndEndIsValid
+          }
+        }}
+        onSubmit={data => handleSubmit(data)}
+        onSubmitFailed={handleSubmitFailed}
+        validateOn="submit"
+      >
+        <AUheading size="xl" level="1">
+          Activity reports
+        </AUheading>
+        <p>Download opportunity report data for your agency or a current list of approved sellers.</p>
+        <ErrorAlert
+          title="An error occurred"
+          model={model}
+          messages={{
+            startDateIsValid: 'Please enter a valid start date',
+            endDateIsValid: 'Please enter a valid end date',
+            startAndEndIsValid: 'Please ensure that the start date is before the end date.'
+          }}
+        />
+        <div className="col-sm-8 col-xs-12">
+          <RadioList
+            className={styles['space-top-2x']}
+            model={`${model}.reportType`}
+            name="reportType"
+            id="reportType"
+            label="Choose the type of report you would like to download for your agency."
+            onChange={() => {
+              this.setState({
+                acceptEnabled: false
+              })
+            }}
+            options={[
+              {
+                value: 'sellersCatalogue',
+                label: 'Current approved sellers catalogue'
+              },
+              {
+                value: 'sellerResponses',
+                label: 'Seller responses to your briefs'
+              },
+              {
+                value: 'specialist',
+                label: 'Digital specialist opportunities'
+              },
+              {
+                value: 'atm',
+                label: 'Ask the market opportunities'
+              },
+              {
+                value: 'rfx',
+                label: 'RFx opportunities'
+              },
+              {
+                value: 'training',
+                label: 'Training opportunities'
+              }
+            ]}
+          />
+          {this.props[model].reportType === 'sellersCatalogue' && (
+            <React.Fragment>
+              <div>
+                <AUcheckbox
+                  className={styles['space-top-bottom-2x']}
+                  id="cb-declaration1"
+                  name="procurementAccessCB"
+                  checked={this.state.acceptEnabled}
+                  onClick={e => {
+                    this.setState({
+                      acceptEnabled: e.target.checked
+                    })
+                  }}
+                  label="I am authorised to access procurement reporting documents across my organisation and I will store, transmit and use the report in line with my agency's privacy and data policies."
+                />
+                <AUbutton type="submit" disabled={!this.state.acceptEnabled}>
+                  Download reports
+                </AUbutton>
+              </div>
```

Comment:
Why are there two submit buttons and checkboxes?

Reply:
removed